NationStatesAPIBot/NationStatesAPIBot/Services/RecruitmentService.cs

```diff
@@ -53,6 +53,11 @@ public RecruitmentService(ILogger<RecruitmentService> logger, IOptions<AppSettin
             _apiService = apiService;
             _dumpDataService = dumpDataService;
             defaulEventId = LogEventIdProvider.GetEventIdByType(LoggingEvent.APIRecruitment);
+            _rnd = new Random();
+            if (_config.EnableRecruitment)
+            {
+                RecruitmentStatus = "Disabled";
+            }
```

Comment:
If EnableRecruitment status is Disabled. That doesn't seem right when just reading it... if it's not a mistake and there's a particular reason for it, could you explain it perhaps in a comment?

Reply:
It is an issue it should be !_config.EnableRecruitment. Thanks for pointing out.